app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/model/CharacterRealm.kt

```diff
@@ -0,0 +1,16 @@
+package com.puzzlebench.clean_marvel_kotlin.domain.model
+
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+
+open class CharacterRealm(
+    @PrimaryKey var id: Int = DEFAULT_ID,
+    var name: String? = DEFAULT_STRING,
```

Comment:
it is necessary that the variable be nullable? pls check this.

Reply:
No, its not necessary. I will change this in next commit!